README.md

```diff
@@ -1,70 +1,63 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Project Name
 
-## Available Scripts
+> My Portfolio
+-It's project built using HTML, CSS3 and react to see how github flow works.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+HTML5
+- CSS3
+- React JS
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Getting Started
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
- [ ] Hi @bigmosi, kindly get rid of these instructions. There are just directives on how to setup up your project locally.